frontend-opal/src/components/PortfolioContainer/AllocationInfo/AllocationInfo.js

```diff
@@ -27,6 +28,14 @@ const AllocationInfo = ({ pieData, pieColor, legendData }) => (
   </View>
 );
 
+const mapStateToProps = state => ({
+  pieData: !state.portfolio[0] ? [] : state.portfolio[0].pieData,
```

Comment:
@WenyuD if portfolio is just a plain object instead of an array (it won't have multiple elements anyway), you can use that as an initial state, so here you don't need to check if there's [0] indexed element in it, you can just use `state.portfolio.pieData`